lib/airbrake.rb

```diff
@@ -19,3 +19,28 @@
 require 'airbrake/resque/failure' if defined?(Resque)
 require 'airbrake/sidekiq/error_handler' if defined?(Sidekiq)
 require 'airbrake/delayed_job/plugin' if defined?(Delayed)
+
+##
+# This module defines the Airbrake API and serves as a namespace for other
+# classes and modules.
+##
+module Airbrake
+  class << self
+    ##
+    # Allows users to add their own notice's builders. For example, if it's needed to
```

Comment:
Could you paraphrase this paragraph so it looks more general? Just mention that with help of this you can attach additional information from the rack environment/request.